registry/lombard/calldata-lbtc-redeem-holesky.json

```diff
@@ -0,0 +1,1494 @@
+{
+  "$schema": "../../specs/erc7730-v1.schema.json",
+  "context": {
+    "$id": "LBTC",
+    "contract": {
+      "deployments": [
+        {
+          "chainId": 17000,
+          "address": "0xED7bfd5C1790576105Af4649817f6d35A75CD818"
+        }
+      ],
+      "abi": [
```

Comment:
Not sure if we need the entire `abi` here, or maybe all we need is just this?

```
{
  "type": "function",
  "name": "redeem",
  "inputs": [
    {
      "name": "scriptPubkey",
      "type": "bytes",
      "internalType": "bytes",
      "components": null,
      "indexed": null,
      "unit": null
    },
    {
      "name": "amount",
      "type": "uint256",
      "internalType": "uint256",
      "components": null,
      "indexed": null,
      "unit": null
    }
  ],
  "outputs": [],
  "stateMutability": "nonpayable",
  "constant": null,
  "payable": null,
  "gas": null,
  "signature": null
},
```

Reply:
Great, I've removed the rest in this commit: https://github.com/LedgerHQ/clear-signing-erc7730-registry/pull/153/commits/aea6d671f24287dad4afc3942ccb27f7ef7da590